{"stability":{"stable since":"1.12.0"},"apiName":"std::sync::mpsc::RecvTimeoutError","apiType":"enum","apiDocumentation":{"apiDescription":["This enumeration is the list of possible errors that made recv_timeout unable to return data when called. This can occur with both a channel and a sync_channel."]},"enum variants":["Timeout","This channel is currently empty, but the Sender(s) have not yet\ndisconnected, so data may yet become available.\n","Disconnected","The channel's sending half has become disconnected, and there will\nnever be any more data received on it.\n"],"trait implementations":[{"name":"impl std::cmp::PartialEq for std::sync::mpsc::RecvTimeoutError","stable since":"1.0.0"},{"name":"impl std::cmp::Eq for std::sync::mpsc::RecvTimeoutError","stable since":"1.0.0"},{"name":"impl std::clone::Clone for std::sync::mpsc::RecvTimeoutError","stable since":"1.0.0"},{"name":"impl std::marker::Copy for std::sync::mpsc::RecvTimeoutError","stable since":"1.0.0"},{"name":"impl std::fmt::Debug for std::sync::mpsc::RecvTimeoutError","stable since":"1.0.0"},{"name":"impl std::fmt::Display for std::sync::mpsc::RecvTimeoutError","stable since":"1.15.0"},{"name":"impl std::error::Error for std::sync::mpsc::RecvTimeoutError","stable since":"1.15.0"},{"name":"impl std::convert::From<std::sync::mpsc::RecvError> for std::sync::mpsc::RecvTimeoutError","stable since":"1.24.0"},{"name":"impl std::marker::Send for std::sync::mpsc::RecvTimeoutError","stable since":"1.0.0"},{"name":"impl std::marker::Sync for std::sync::mpsc::RecvTimeoutError","stable since":"1.0.0"}]}